distributed/http/worker/tests/test_worker_http.py

```diff
@@ -56,6 +58,27 @@ async def test_prometheus(c, s, a):
     await fetch_metrics(a.http_server.port, prefix="dask_worker_")
 
 
+@pytest.fixture
+def prometheus_not_available():
+    import sys
+
+    with mock.patch.dict("sys.modules", {"prometheus_client": None}):
+        del sys.modules["distributed.http.worker.prometheus"]
+        yield
+
+
+@gen_cluster(client=True, nthreads=[])
+async def test_metrics_when_prometheus_client_not_installed(
+    c, s, prometheus_not_available
+):
+    with mock.patch.dict("sys.modules", {"prometheus_client": None}):
+        import distributed
```

Comment:
`distributed` is certainly already imported by this point to run the `gen_cluster`